en/orm/saving-data.rst

```diff
@@ -610,7 +610,7 @@ data needs to be restructured so it can be put into the right fields.
 
 The ``Model.beforeMarshal`` event is triggered just at the start of the
 validation process, one of the reasons is that ``beforeMarshal`` is allowed to
-change the validation rules and the saving options, such as the field whitelist.
+change the validation rules and the saving options, such as the field list.
```

Comment:
This run-on sentence needs to be broke up but github won't let me make a multi-line suggestions here.

Reply:
Can do that separately.